src/icon-manifest/manifestReader.ts

```diff
@@ -0,0 +1,30 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as models from '../models';
+import { parseJSON } from '../utils';
+import { extensionSettings } from '../settings';
+
+export class ManifestReader {
+
+  public static iconsDisabled(name: string, isFile: boolean = true): boolean {
+    const iconManifest = this._getIconManifest();
+    const iconsJson = iconManifest && parseJSON(iconManifest) as models.IIconSchema;
+    return !iconsJson || !Reflect.ownKeys(iconsJson.iconDefinitions)
+      .filter(key => key.toString().startsWith(`_${isFile ? 'f' : 'fd'}_${name}`)).length;
+  }
+
+  public static folderIconsDisabled(func: (iconsJson: models.IIconSchema) => boolean): boolean {
+    const iconManifest = this._getIconManifest();
+    const iconsJson = iconManifest && parseJSON(iconManifest) as models.IIconSchema;
+    return !iconsJson || !func(iconsJson);
+  }
+
+  private static _getIconManifest(): string {
```

Comment:
I know this is an existing function but I didn't see the underscore until now. Why the use of this syntax if we already have the private modifier?

Reply:
This is what happens when you write code in many projects with different coding styles. I'll fix it as soon as I got the time or you can do that too if everything else is fine.

Reply:
Fixed.
@robertohuertasm In order to avoid same cases in the future how about using naming convention `tslint` rule like they do in [tslint-microsoft-contrib](https://github.com/Microsoft/tslint-microsoft-contrib)?

Reply:
We're already using `tslint:latest`. I guess this library you propose is composed by custom rules. I've never tried it so I don't have any opinion.

Reply:
You are right. Those are custom rules. Tslint latest doesn't have naming convention rules. On the other hand, maybe this is too much of a hassle for such a small group of developers.